src/config/markets.ts

```diff
@@ -147,3 +147,26 @@ export function getMarketListingDate(chainId: number, marketAddress: string): Da
 
   return tokenListing.listingDate;
 }
+
+export const GLV_MARKETS_ENABLED = {
+  [ARBITRUM]: false,
+  [ARBITRUM_GOERLI]: false,
+  [AVALANCHE]: false,
+  [AVALANCHE_FUJI]: true,
+};
+
+export const GLV_MARKETS_TITLE: {
```

Comment:
maybe rename it somehow so that it can store other settings? e.x. abbr `HC` can be stored here as well I think

Reply:
yeah, good idea. we've already storing all appearance data like human readable name and subtitle, guess we can store shortening here as well, instead of calculating it by title